vtr_flow/scripts/python_libs/vtr/log_parse.py

```diff
@@ -23,7 +23,7 @@ class ParsePattern:
     def __init__(self, name, filename, regex_str, default_value=None):
         self._name = name
         self._filename = filename
-        self._regex = re.compile(regex_str)
+        self._regex = re.compile(f'^.*{regex_str}.*$')
```

Comment:
It would be good for you to add a comment saying what this means: look for the specified pattern somewhere in the line, but any characters can occur before and after it.

Reply:
Thanks, Vaughn! It has been added.